src/device/selection.ts

```diff
@@ -440,3 +440,29 @@ export const html = () => {
   const currentHtml = div.innerHTML
   return currentHtml
 }
+
+/** Returns the bounding rectangle for the current browser selection. */
+export const getBoundingClientRect = () => {
+  const selection = window.getSelection()
+
+  if (selection && selection.rangeCount) {
+    return selection.getRangeAt(0).getBoundingClientRect()
+  }
+
+  return null
+}
+
+/** Returns true if the point is within 100 pixels of the browser selection. */
+export const isNear = (x: number, y: number): boolean => {
+  if (!isActive() || isCollapsed()) return false
+
+  const rect = getBoundingClientRect()
+  if (!rect) return false
+
+  const left = rect.left - 100
+  const right = rect.right + 100
+  const top = rect.top - 100
+  const bottom = rect.bottom + 100
+
+  return x >= left && y >= top && x <= right && y <= bottom
```

Comment:
Yes, I was thinking it would be a full width zone. That said, I don't mind the 100px on all sides at all. I think it still accomplishes the goal of preventing gestures from interfering with text selection.

Reply:
I can make the change quickly if you decide you want it to be full width.